core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -31,6 +31,7 @@
     public static final String JDBC_MSSQL = "jdbc-mssql";
     public static final String JDBC_MYSQL = "jdbc-mysql";
     public static final String JGIT = "jgit";
+    public static final String JSCH = "jsch";
```

Comment:
Open question: do we really want to expose it or should it stay internal for now? My inclination would be to make it internal until someone asked for it.

Reply:
I'm not opposed to making it internal. So far jGit and camel-ftp were the only extensions using it. How do I make it internal BTW? Just removing the FeatureBuildItem?

Reply:
That and https://github.com/quarkusio/quarkus/blob/master/extensions/caffeine/runtime/src/main/resources/META-INF/quarkus-extension.yaml#L9 .

Reply:
Should it be internal ? @lburgazzoli any suggestion?

Reply:
The question is basically:
- do we want users to be able to use it directly?
- do we want to expose it in the extension list at startup?

I think, for now, the answers to these questions are both no.

Reply:
I am with @gsmet on this, it doesn't look like this is something that would be used directly by users.